prime-dotnet-webapi/Services/interfaces/IOrganizationService.cs

```diff
@@ -15,7 +15,10 @@ public interface IOrganizationService
         Task<IEnumerable<OrganizationSearchViewModel>> GetOrganizationsAsync(OrganizationSearchOptions search);
         Task<IEnumerable<OrganizationListViewModel>> GetOrganizationsByPartyIdAsync(int partyId);
         Task<Organization> GetOrganizationAsync(int organizationId);
+        Task<Organization> GetOrganizationByPecAsync(string pec);
         Task<int> CreateOrganizationAsync(int signingAuthorityId);
+        Task<Organization> ClaimOrganizationAsync(OrganizationClaimViewModel claimOrganization);
+        Task<bool> OrganizationClaimExistsAsync(OrganizationClaimSearchOptions search);
```

Comment:
These two methods seem like they could be on the `OrganizationClamService`, rather than this ervice